Add GA Events

teodorus-nathaniel opened the pull request:
This GA Event also include `userId` using the address of connected account.

# List of Events
- Connect wallet => Click on connect wallet button
- Change account => Change account to {address}
- Disconnect => Disconnect wallet
- Find Tweet => Click on find tweet button
- Change space id => Change space id to {spaceId}
- Publish => Click on Publish to Subsocial button
- Discord link in energy alert => Click on discord link in energy alert
- Click copy button in energy alert => Click on copy energy command button in energy alert
- Copy energy command => Click on copy energy command button in energy alert
- Click on `Twitter Backups` link => Go to Twitter Backups Space
- Click on `My Saved Tweets` => Go to My Saved Tweets

# Other Changes
- Remove `My Saved Tweet` link if not login.

# To be confirmed
- Do you want to add address as userId for every ga event?
- Do you want to save user `address` in change account event?
- Do you want to save `spaceId` in change space event?

nathankurnia26 committed:
Fix get energy command and only export p4espace getter

nathankurnia26 committed:
Make p4e space default to selected space dropdown

nathankurnia26 committed:
Add ga events to several actions

nathankurnia26 committed:
Hide my saved tweets link if not login

nathankurnia26 committed:
Change ga event for changing account to use address instead of name

olehmell commented:
Additional events:
- Click on my tweet link
- Click on "Enter app" on the landing

nathankurnia26 committed:
Add ga event on enter app

teodorus-nathaniel commented:
> Additional events:
> 
> * Click on my tweet link
> * Click on "Enter app" on the landing

For the on my tweet link, I have added ga event on that, just forgot to add it into the list. (I have added in the list now)
